.github/workflows/build-accumulo.yml

```diff
@@ -80,20 +80,22 @@ jobs:
     - name: Log in to the Container registry
       uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
       with:
-        registry: ${{ env.REGISTRY }}
-        username: ${{ env.USER_NAME }}
-        password: ${{ env.ACCESS_TOKEN }}
+        registry: ${{ env.GITHUB_REGISTRY }}
+        username: ${{ env.GITHUB_USER_NAME }}
+        password: ${{ env.GITHUB_ACCESS_TOKEN }}
 
     - name: Update DataWave Dependency Version
       run: |
 
         export JAVA_HOME="$DATAWAVE_JAVA"
         cd "$GITHUB_WORKSPACE/datawave"
-        mvn -s "$GITHUB_WORKSPACE/datawave/.github/workflows/settings.xml" versions:set-property -Dproperty=version.accumulo -DnewVersion=${{ steps.get-accumulo-version.outputs.accumuloVersion }} -DgenerateBackupPoms=false
+        cp $GITHUB_WORKSPACE/.github/workflows/settings.xml ~/.m2/settings.xml
```

Comment:
Why copy the file instead of just using the -s? I am not necessarily opposed just curious why.